packages/scheduler/cleanup.py

```
@@ -110,13 +114,37 @@ async def nuclear_cleanup(db: Any) -> dict[str, int]:
+                to_delete = job_ids[1:]
+                if to_delete:
+                    r3 = await col.delete_many({"job_id": {"$in": to_delete}})
+                    summary["deduped"] += r3.deleted_count if hasattr(r3, 'deleted_count') else len(to_delete)
+        except Exception as exc:  # noqa: BLE001
+            log.debug("Nuclear cleanup dedup failed (non-fatal — SQLite may not support $aggregate): %s", exc)
+
+        # Count total remaining
+        try:
+            summary["total"] = await col.count_documents({})
+        except Exception:  # noqa: BLE001
+            pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)